Automattic/jetpack
# VideoPress: Improving feedback for local media upload

Improving feedback while waiting for local media metadata fetching.

Closes #25942 

#### Changes proposed in this Pull Request:

* Include loading state while verifying local media.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Start `Jetpack VideoPress`
* Upload local video through `VideoPress` block.
* You should see `Loading...` screen before verifying local media.
